src/decoder/mod.rs

```diff
@@ -185,6 +185,28 @@ impl Default for Limits {
     }
 }
 
+/// Metadata for a ScanImage BigTiff file. The first four bytes of metadata
+/// are always SI's magic number which identifies the image as an SI image,
+/// and thus they're left out of the struct. Requesting the ScanImageMetadata
+/// struct from the Decoder via `decoder.scanimage_metadata()` may result in 
+/// a None if the file is not a ScanImage BigTiff.
+#[derive(Debug, Clone)]
+pub struct ScanImageMetadata {
+    /// Bytes 4 to 8 of the static metadata section
+    pub tif_version: u32,
+    /// Non-varying frame data
+    pub nonvar_frame_data: String,
+    /// ROI group data
+    pub roi_group_data: String,
+}
```

Comment:
This should have a private member `_private: ()` to ensure that further fields can be added without a breaking change. I do not know enough about the format to judge if this is likely _but_ without an open standard I'd rather be cautious than require a major bump for adding/changing support for the metadata.